larq_compute_engine/tflite/kernels/bconv2d.cc

```diff
@@ -213,10 +212,9 @@ TfLiteStatus Prepare(KernelType kernel_type,
       return kTfLiteError;
     }
 
-    // TODO: more intelligent selection of the parameters `bitpacking_bitwidth`
-    //       and `bitpack_before_im2col` based on benchmarking results
-    //       (as in https://github.com/larq/compute-engine/issues/290).
-    conv_params->bitpacking_bitwidth = default_bitpacking_bitwidth;
```

Comment:
Should we also remove `bitpacking_bitwidth` from `TfLiteBConv2DParams`?

https://github.com/larq/compute-engine/blob/2a4d5132104cf4dea3debb44eed96b76d1c40531/larq_compute_engine/tflite/kernels/bconv2d_params.h#L80

Reply:
Good spot, thanks :+1: